<launch>
  <rosparam file="$(find forcecontrol)/test/hardware.yaml" command="load"/>
  <rosparam file="$(find forcecontrol)/config/controllers.yaml" command="load"  />
  <node pkg="forcecontrol" name="force_control_hw" type="forcecontrol_node"  output="screen" />
</launch>
